doc/user_manual/files.tex

```diff
@@ -29,6 +29,16 @@ \section{Files}
   directly use this attribute, but it is availabe in the CodeInterface.
   If not provided, defaults to True.
   %
+  
+  \item \xmlAttr{subDirectory}, \xmlDesc{optional string attribute}, sub-directory
+  that should be created in the perturbation process. If specified, the file will be
+  placed in the subdirectory. For example, in a \textit{MultiRun} step using a \textit{MonteCarlo}
```

Comment:
Is the Sampler used relevant here? This seems like extra information not used in determining the path.